guides/advancingGuides/advanceGuideUsingDynamicPermalinkButton/UsingClickableImage/html_for_clickable_image.html

```diff
@@ -0,0 +1,13 @@
+<!--Exposing hidden metadata from the guides HTML, you can use visitor or account metadata below depending on what you need in the Permalink URL-->
+<span class="_pendo-hidden-account_"><%= metadata.account.name %></span>
+<span class="_pendo-hidden-visitor"><%= metadata.visitor.email %></span>
```

Comment:
should be `_pendo-hidden-visitor_` -- missing ending _

Reply:
Sure will make the change.

Reply:
Implented the changes.